### PR TITLE
fix(mainnet): update powergate docs for mainnet

### PR DESCRIPTION
**Pull request by andrewxhill**
Signed-off-by: Andrew Hill <andrew@textile.io>

### CODE REVIEW

File: docs/powergate/mainnet.md [hunk 1]
```diff
@@ -46,27 +46,27 @@ mined block in the past	{"block-time": "2009-01-01T04:44:30.000Z",\
...
 
 You should edit the `docker/docker-compose.yaml` file adding two lines:
 ![image](https://user-images.githubusercontent.com/6136245/93375329-6383fd80-f82e-11ea-9850-2970f30a793c.png)
 
-Then run `make up`. If you inspect the Lotus node with `docker logs testnet_lotus_1`, you will see a message that is importing chain information, and a bunch of Badger compaction messages. After the importing is done, the Lotus node will continue to sync as usual showing the typical logs. At this point you should `make down` (you don't need to wait to full syncing), revert `docker/docker-compose.yaml` to the original content, and `make up` again.
+Then run `make up`. If you inspect the Lotus node with `docker logs mainnet_lotus_1`, you will see a message that is importing chain information, and many Badger compaction messages. After the importing is done, the Lotus node will continue to sync as usual showing the typical logs. At this point you should `make down` (you don't need to wait to full syncing), revert `docker/docker-compose.yaml` to the original content, and `make up` again.
```
Comment (andrewxhill) [inline docs/powergate/mainnet.md:62]: didn't test this live, flagging for a check

Comment (jsign) [reply docs/powergate/mainnet.md:62]: Uhm, no. Needs some extra change un the Makefile in Powergate 'docker' folder.
Can do soon.

**Review by jsign — APPROVED** [review]
LGTM. I can change what mentioned in the comment and merge both at the same time later.

**Review by asutula — APPROVED** [review]

File: docs/powergate/mainnet.md (hunk 1 above)
Comment (andrewxhill) [reply docs/powergate/mainnet.md:62]: okay, so will push this to docs for the time-being. let me know if you want a quick pr there

Comment (jsign) [reply docs/powergate/mainnet.md:62]: Targeted [here](https://github.com/textileio/powergate/pull/681).

**Pull request merged**